query/src/storages/fuse/meta/versions.rs

```diff
@@ -0,0 +1,103 @@
+//  Copyright 2022 Datafuse Labs.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+use std::marker::PhantomData;
+
+use common_datablocks::DataBlock;
+use common_exception::ErrorCode;
+
+use crate::storages::fuse::meta::common::Versioned;
+use crate::storages::fuse::meta::v0;
+use crate::storages::fuse::meta::v1;
+
+pub const CURRNET_SEGMETN_VERSION: u64 = v1::SegmentInfo::VERSION;
```

Comment:
Guess `CURRNET_*` should be `CURRENT_*`

Reply:
Technically, these constants could be removed. these are the legacy code to ensure that for each type of table meta type (e.g. SegmentInfo), there is only one numeric number has been bound to it (the `impl Version<n> for SegmentInfo`)